frontend/src/components/common/header/header.tsx

```diff
@@ -1,34 +1,51 @@
 import defaultUserAvatar from 'assets/img/avatar-default.svg';
 import logo from 'assets/img/logo.svg';
+import { AppRoute } from 'common/enums/enums';
 import { FC } from 'common/types/types';
-import { Image } from 'components/common/common';
-import { useState } from 'hooks/hooks';
+import { Button, Image } from 'components/common/common';
+import { useAppSelector, useNavigate, useState } from 'hooks/hooks';
 
 import { Popup } from './components/components';
 import styles from './styles.module.scss';
 
 const Header: FC = () => {
   const [isMenuPopupVisible, setIsMenuPopupVisible] = useState<boolean>(false);
+  const { user } = useAppSelector((state) => state.auth);
+  const navigate = useNavigate();
 
   const handlePopupOper = (): void =>
     setIsMenuPopupVisible(!isMenuPopupVisible);
 
+  const handleNavigateToLogIn = (): void => navigate(AppRoute.SIGN_IN);
+  const handleNavigateToSignUp = (): void => navigate(AppRoute.SIGN_UP);
+
   return (
     <header>
       <div className={styles.headerWrapper}>
         <div className={styles.logoWrapper}>
           <Image width="150" height="94" src={logo} alt="logo" />
         </div>
         <div className={styles.userWrapper}>
-          <button onClick={handlePopupOper} className={styles.button}>
-            <Image
-              width="50"
-              height="50"
-              src={defaultUserAvatar}
-              alt="user avatar"
-              isCircular
-            />
-          </button>
+          {!user ? (
+            <div className={styles.buttonsWrapper}>
+              <Button
+                onClick={handleNavigateToLogIn}
+                label="Log In"
+                color="grey"
+              />
```

Comment:
```suggestion
              <Button
                onClick={handleNavigateToLogIn}
                label="Log In"
                color="grey"
                to={AppRoute.ROUT}
              />
```
modify button component. if we provide the `to` prop it should render a link instead

Reply:
done